pkg/errs/errors.go

```diff
@@ -0,0 +1,8 @@
+package errs
+
+import "errors"
+
+var DbError = errors.New("unexpected database error")
+var ContentType404 = errors.New("content type not found")
+var Column404 = errors.New("column not found")
```

Comment:
You might want to name these errors with an `Error` or `Err` prefix like so: `ErrorColumnNotFound`

Reply:
Done, check the last commit